packages/test/test-utils/src/loaderContainerTracker.ts

```diff
@@ -125,6 +100,31 @@ export class LoaderContainerTracker implements IOpProcessingController {
 			};
 			containerWithClone.clone = patch(containerWithClone.clone);
 		}
+
+		// ignore summarizer
+		if (
+			!container.deltaManager.clientDetails.capabilities.interactive &&
+			!this.syncSummarizerClients
+		) {
+			return;
+		}
+
+		// don't add container that is already tracked
+		if (this.containers.has(container)) {
```

Comment:
We probably want to move this check up before patching (don't want to repeatedly wrap the clone function.

Reply:
done